### ISSUE TITLE
Subtractive lighting not supported

### ISSUE THREAD

**Issue by Fejuto** [Cyanilux/URP_ShaderGraphCustomLighting#13]
Reporting a bug/missing feature:

When using Subtractive lighting https://drive.google.com/uc?id=1r5fO4GJOwFskzgW2L4UOHaXcUEYRrB36, the toon shader from this Github doesn't receive the Subtractive shadow as demonstrated here https://drive.google.com/uc?id=1EWzjEFLrqJQTN0HGQgcFCnqUr0LwEI4A

My project uses Subtractive lighting so would be great if this feature could be supported!

**Comment by Cyanilux** [comment]
Hey there! I've updated the main (URP v12+) branch to add support for this~
(I've also added the `SubtractiveGI_float` function to the older branch (for v8 - v11) in case anyone needs it, but that's probably as much as I'll do there)
I'm not that familiar with Baked GI (hence why this wasn't included earlier) so hopefully I've handled it correctly (seems to work okay on my end now at least!). But let me know if there's any issues.
Thanks!
